website/content/docs/release-notes/consul/v1_19_x.mdx

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+page_title: 1.19.x
+description: >-
+  Consul release notes for version 1.19.x
+---
+
+# Consul 1.19.0
+
+We are pleased to announce the following Consul updates.
+
+## Release highlights
+
+- TBD
+
+## What's deprecated
+
+- **Snapshot Agent (Enterprise):** Starting with this release, top level single snapshot destinations `local_storage`, `aws_storage`, `azure_blob_storage`, and `google_storage` in snapshot agent configuration files are deprecated. Use the `backup_destinations` config object instead.
```

Comment:
Could you add a link to the new backup destinations stanza please?

Reply:
Good call! Fixed in https://github.com/hashicorp/consul/pull/20848/commits/634e29107611fbf3a9fac084603e527385f13a6b